mlxtend/regressor/stacking_regression.py

```diff
@@ -111,6 +111,8 @@ def fit(self, X, y):
             n_features is the number of features.
         y : array-like, shape = [n_samples] or [n_samples, n_targets]
             Target values.
+        sample_weight : array-like, shape = [n_samples], optional
+            Sample weights.
```

Comment:
Could you additionally specify that these are used by both the level-1 and meta-regressors?

Reply:
Sure.  And the meta regressor too.

Reply:
Yeah, maybe as doc string sth like

Sample weights passed as `sample_weights` to each regressor in the `regressors` list as well as the `meta_regressor`.

The rest of the PR looks great btw! Thanks!